java-numbers/src/main/java/com/baeldung/prime/BruteForcePrimeChecker.java

```diff
@@ -1,16 +1,14 @@
-package com.baeldung.algorithms.primechecker;
+package com.baeldung.prime;
```

Comment:
What is your reasoning for the package name change?

If it is something like "readability" or "cosmetics", then let's try and avoid this for this improvement. (While I do think these changes are great, they can be done in a different ticket)